### ISSUE TITLE
Tag Parsing

### ISSUE THREAD

**Issue by jacksonhyde** [jacksonhyde/InkyBot#4]
There is currently no parsing of [tags](https://github.com/inkle/ink/blob/master/Documentation/WritingWithInk.md#tags). This is particularly apparent in The Intercept, as it's a tag that's used to signify the End of the Game.

There's a lot of scope to do lots of really cool things with tags, but for now I think the focus should be on delivering just the following:

- Create an API/Service for parsing tags
- Create a handler for the 'End Game' tag

**Comment by rondaa** [comment]
Here are the tags ink.js handles

- Title
- Author
- Theme
- Image
- Class
- End

Theme (light/dark) doesn't make sense to support, and I doubt we can support Class (custom css) at this stage. The other ones are rather simple, with the caveat that we don't allow the bot to serve its own images.

**Comment by jacksonhyde** [comment]
Yeah we definitely want to support those global tags - I agree that class and theme can be skipped though.

I think we also need to think about how authors might want to use tags in this manner:
https://github.com/inkle/ink/blob/master/Documentation/RunningYourInk.md#marking-up-your-ink-content-with-tags
